doc/content/guides/getting-started/console/create-application/_index.md

```diff
@@ -18,11 +18,11 @@ Your application will be created and you will be redirected to the application o
 
 {{< figure src="application-overview.png" alt="Application overview" >}}
 
-You can now use the builtin MQTT server and webhooks to receive uplink traffic and send downlink traffic. End devices are created within applications. 
+You can now use the builtin MQTT server and webhooks to receive uplink traffic and send downlink traffic. End devices are created within applications.
 
 ### Link application
 
-If you haven't unchecked the **Link automatically** checkbox during creation, your device will be automatically linked to the Application Server. You can skip this section in this case.
+If you did not uncheck the **Link automatically** checkbox during creation, your application will be automatically linked to the Application Server. You can skip this section in this case.
```

Comment:
Why change `haven't unchecked` to `did not uncheck`? They are grammatically equivalent.

Reply:
This is mostly because of our "Avoid shortening" rule. There may be more instances where we do this I think.